cmd/starter/c/starter.c

```diff
@@ -843,29 +831,55 @@ void do_exit(int sig) {
     exit(1);
 }
 
+/*
+ * clearenv set environ pointer to NULL, while it works
```

Comment:
Here we say `clearenv`, line 840 the function is called `cleanenv`. That's probably just a typo

Reply:
Oops ! fixed